stdlib/public/SDK/Foundation/Data+DataProtocol.swift

```diff
@@ -0,0 +1,46 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2018 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+
+extension Data : MutableDataProtocol, MutableContiguousCollection {
+    // -- ContiguousCollection -------------------------------------------------
```

Comment:
As a style thing I think we can likely get rid of dividers like these from my initial revision

Reply:
ok will do